bcipy/task/data.py

```diff
@@ -249,7 +270,7 @@ def as_dict(self,
             'task': self.task,
             'mode': self.mode,
             'series': series_dict,
-            'total_time_spent': self.total_time_spent,
+            'total_time_spent': round(self.total_time_spent, 2),
```

Comment:
I'd pull this up as a instance variable. `self.time_spent_precision = 2` or use the same as `self.precision`?